yarn.lock

```diff
@@ -0,0 +1,99 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.

```

Comment:
I installed `@types/imap` using Yarn, if you prefer to stick with NPM, I'll just remote the lock file from the PR.